### PR TITLE
TS-130 - Fixing a few small issues

### PR DESCRIPTION
**Pull request by mcromie92**
- Removing mandatory holiday logic from the front end  - this is going to be added in version 2
- Removing cancel button from approved holidays
- Fix issues that I broke with a merge earlier this morning

### CODE REVIEW

File: src/components/BookingModal/ModalStatusBanner/index.jsx [hunk 1]
```diff
@@ -70,7 +71,8 @@ class ModalStatusBanner extends Component {
           </div>
         )}
         <div>
-          {!isCancelled && (
+          {!isCancelled &&
+            !isApproved && (
```
Comment (d1wilko) [inline src/components/BookingModal/ModalStatusBanner/index.jsx:75]: this is cleaner on one line `{!isCancelled && !isApproved && (`

**Review by d1wilko — CHANGES_REQUESTED** [review]
just one change for me

File: src/components/BookingModal/ModalStatusBanner/index.jsx (hunk 1 above)
Comment (mcromie92) [reply src/components/BookingModal/ModalStatusBanner/index.jsx:75]: @d1wilko I think eLint is saving it that way

Comment (d1wilko) [reply src/components/BookingModal/ModalStatusBanner/index.jsx:75]: weird... but ok

**Review by d1wilko — APPROVED** [review]

**Pull request merged**